src/DataWrangling/ECCO/ECCO_metadata.jl

```diff
@@ -262,10 +265,13 @@ function download_dataset(metadata::ECCOMetadata; url = urls(metadata))
                            and setting your ECCO_USERNAME and ECCO_PASSWORD." * instructions_msg
                     throw(ArgumentError(msg))
                 end
-
+		println("Downloading ECCO data: $(metadatum.name)")
                 Downloads.download(fileurl, filepath; downloader, progress=download_progress)
-            end
+	    end
         end
+	if !missing_files
+            println("Note: ECCO $(metadata.name) data is in $(metadata.dir).")
```

Comment:
```suggestion
            @info "Note: ECCO $(metadata.name) data is in $(metadata.dir)."
```
we typically use `@info` for printed information (it just has a nicer formatting 😄 )